tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/packaging/PackagingTestCase.java

```diff
@@ -286,39 +301,27 @@ public void testBuildAndRunModuleWithPeriod() throws BallerinaTestException {
         balClient.runMain("add", new String[] { moduleName }, envVariables, new String[] {}, new LogLeecher[] {},
                 projectPath.toString());
 
-        Assert.assertTrue(Files.exists(projectPath.resolve("src").resolve(moduleName)));
-        Assert.assertTrue(Files.isDirectory(projectPath.resolve("src").resolve(moduleName)));
+        Path modulePath = projectPath.resolve("src").resolve(moduleName);
+        Assert.assertTrue(Files.exists(modulePath));
+        Assert.assertTrue(Files.isDirectory(modulePath));
```

Comment:
We can get rid of asserting whether file exists because Files.isDirectory(modulePath) will fail anyway if the file doesn't exist. WDYT?